src/ape/managers/networks.py

```diff
@@ -54,8 +55,8 @@ def ecosystems(self) -> Dict[str, EcosystemAPI]:
                 data_folder=self.config_manager.DATA_FOLDER / plugin_name,
                 request_header=self.config_manager.REQUEST_HEADER,
             )
-            ecosystem_config = self.config_manager.get_config(plugin_name)
-            default_network = ecosystem_config.default_network  # type: ignore
+            ecosystem_config = self.config_manager.get_config(plugin_name).dict()
```

Comment:
I'm super good with this, unless we want to change the way `get_config` works to return the dict?

Reply:
I think in some contexts, the class is nice, like within the plugin itself. But here in the managers, we can't assume anything about the class